bedrock/firefox/templates/firefox/private-browsing.html

```diff
@@ -0,0 +1,136 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import fxfamilynav with context %}
+
+{% extends "/firefox/base-resp.html" %}
+
+{% block page_title %}{{ _('Keep your Firefox in sync') }}{% endblock %}
```

Comment:
No, I missed that (copied from the sync page). I'm not sure what the title of this page should be...